### PR TITLE
Auto archive documentation

### PR DESCRIPTION
**Pull request by luchodev**
Resolves https://linear.app/hashboard/issue/HB-8414/add-documentation-for-auto-trashing-feature

Adding auto archive documentation:

Screenshot 2024-03-25

### CODE REVIEW

File: pages/docs/project-management/auto-archive.mdx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Auto archive allow you to automatically move to trash resources that have not been viewed based on your settings.
+
+## Settings
+
+- Days: Set the time frame for automatically moving unused resources to trash. By default, resources are moved after 30 days of inactivity.
+- Exclude resources based on status: You can control which resources shouldn't be moved to trash based on their status (Exploratory, In Progress, Verified, Archive, No status).
+- Exclude resources based on User owner: You can control which resources shouldn't be moved to trash base on their user owner.
+- Exclude resources based on type: You can control which resources shouldn't be moved to trash based on their type, currently the resources allowed to be auto-trased are: Saved explorations, Dashboards and Collections.
+
+## Test
```
Comment (nstokoe) [inline pages/docs/project-management/auto-archive.mdx:18]: Let's rename this Preview like in the app

File: pages/docs/project-management/auto-archive.mdx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+You can validate which resources meet your configured settings. Clicking the 'Test' button will display those resources. Additionally, you have the option to manually move those resources to trash.
+
+<Callout type="info">
+  The automatic process occurs every night at midnight.
```
Comment (nstokoe) [inline pages/docs/project-management/auto-archive.mdx:23]: 2am EST

**Review by nstokoe — COMMENTED** [review]
Can we add a section about built-in rules
- only empty collections get auto trashed
- saved explorations and dashboards with a scheduled report won't get auto trashed
- data ops resources are excluded.
- saved explorations used in a dashboard won't get trashed

File: pages/docs/project-management/auto-archive.mdx (hunk 1 above)
Comment (luchodev) [reply pages/docs/project-management/auto-archive.mdx]: Done

File: pages/docs/project-management/auto-archive.mdx (hunk 2 above)
Comment (luchodev) [reply pages/docs/project-management/auto-archive.mdx]: Done

File: pages/docs/project-management/auto-archive.mdx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+# Auto archive
+
+Auto archive allow you to automatically move to trash resources that have not been viewed based on your settings.
+
+## Settings
+
+- Days: Set the time frame for automatically moving unused resources to trash. By default, resources are moved after 30 days of inactivity.
+- Exclude resources based on status: You can control which resources shouldn't be moved to trash based on their status (Exploratory, In Progress, Verified, Archive, No status).
+- Exclude resources based on User owner: You can control which resources shouldn't be moved to trash base on their user owner.
```
[the remark this answers is not in the document]
Comment (luchodev) [reply pages/docs/project-management/auto-archive.mdx]: Done

File: pages/docs/project-management/auto-archive.mdx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+import { Callout } from "nextra-theme-docs";
+
+# Auto archive
+
+Auto archive allow you to automatically move to trash resources that have not been viewed based on your settings.
+
+## Settings
+
+- Days: Set the time frame for automatically moving unused resources to trash. By default, resources are moved after 30 days of inactivity.
+- Exclude resources based on status: You can control which resources shouldn't be moved to trash based on their status (Exploratory, In Progress, Verified, Archive, No status).
```
[the remark this answers is not in the document]
Comment (luchodev) [reply pages/docs/project-management/auto-archive.mdx]: Done

File: pages/docs/project-management/auto-archive.mdx [hunk 5]
```diff
@@ -0,0 +1,32 @@
+---
+description: Auto archive allow you to automatically move to trash resources that have not been viewed based on your settings.
+---
+
+import { Callout } from "nextra-theme-docs";
+
+# Auto archive
+
+Auto archive allow you to automatically move to trash resources that have not been viewed based on your settings.
```
Comment (nstokoe) [inline pages/docs/project-management/auto-archive.mdx:9]: "Auto archive allow you to automatically move resources to the trash that have not been recently viewed."

File: pages/docs/project-management/auto-archive.mdx [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+## Settings
+
+- Days: Set the time frame for automatically moving unused resources to trash. By default, resources are moved after 30 days of inactivity.
```
Comment (nstokoe) [inline pages/docs/project-management/auto-archive.mdx:13]: Let's be more specific the just `inactivity`. Maybe something like "after 30 days without any views by users of your project."

File: pages/docs/project-management/auto-archive.mdx [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+- Exclude resources based on [status](/docs/project-management/resource-metadata#status): You can control which resources shouldn't be moved to trash based on their status (Exploratory, In Progress, Verified, Archive, No status).
```
Comment (nstokoe) [inline pages/docs/project-management/auto-archive.mdx:14]: "the trash"

File: pages/docs/project-management/auto-archive.mdx [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+- Exclude resources based on User [owner](/docs/project-management/resource-metadata#owner): You can control which resources shouldn't be moved to trash base on their user owner.
+- Exclude resources based on type: You can control which resources shouldn't be moved to trash based on their type, currently the resources allowed to be auto-trased are: Saved explorations, Dashboards and Collections.
+
+## Preview
+
```
Comment (nstokoe) [inline pages/docs/project-management/auto-archive.mdx:19]: `You can validate which resources meet your configured settings clicking the 'Preview' button.`

File: pages/docs/project-management/auto-archive.mdx [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+  The automatic process occurs every night at 2am EST.
+</Callout>
+
+## Built-in rules
+
+- Only empty Collections get auto-trashed.
+- Saved explorations and dashboards with a scheduled report won't get auto trashed.
+- Saved explorations used in a dashboard won't get trashed.
+- Resources publicly shared won't get trashed.
+- DataOps resources are excluded.
```
Comment (nstokoe) [inline pages/docs/project-management/auto-archive.mdx:32]: `DataOps resources won't get trashed.`

Comment (luchodev) [reply pages/docs/project-management/auto-archive.mdx]: 👍

File: pages/docs/project-management/auto-archive.mdx (hunk 8 above)
Comment (luchodev) [reply pages/docs/project-management/auto-archive.mdx:19]: 👍

File: pages/docs/project-management/auto-archive.mdx (hunk 7 above)
Comment (luchodev) [reply pages/docs/project-management/auto-archive.mdx]: 👍

File: pages/docs/project-management/auto-archive.mdx (hunk 6 above)
Comment (luchodev) [reply pages/docs/project-management/auto-archive.mdx]: 👍

File: pages/docs/project-management/auto-archive.mdx (hunk 5 above)
Comment (luchodev) [reply pages/docs/project-management/auto-archive.mdx]: 👍

**Review by nstokoe — APPROVED** [review]

**Pull request merged**